### PR TITLE
Kses - Part 10

### PR DESCRIPTION
**Pull request by hoseinrafiei**
This is related to https://github.com/eventespresso/event-espresso-core/issues/3738 issue.

Applied `wp_kses` function to some more variables related to iCal, Invoice, Receipt, Error, Debug and Exception display.

Checklist: https://trello.com/c/trYdO3sW/1161-ee4-core-testing-checklist-pr-3834

### CODE REVIEW

File: core/exceptions/ExceptionStackTraceDisplay.php [hunk 1]
```diff
@@ -77,7 +78,7 @@ protected function displayException(Exception $exception)
             echo wp_json_encode(array('error' => $styles . $output . $scripts));
             exit();
         }
-        echo $styles, $output, $scripts; // already escaped
+        echo $styles, wp_kses($output, AllowedTags::getWithFormTags()), $scripts;
```
Comment (tn3rb) [inline core/exceptions/ExceptionStackTraceDisplay.php:81]: any time you find css or js that's being dumped directly to the DOM like this, plz create an issue for fixing it, thnx

**Review by tn3rb — APPROVED** [review]
looks great 💯

File: core/exceptions/ExceptionStackTraceDisplay.php (hunk 1 above)
Comment (hoseinrafiei) [reply core/exceptions/ExceptionStackTraceDisplay.php]: I created a new issue for this instance and I will add the rest to the created issue: https://github.com/eventespresso/event-espresso-core/issues/3839

**Comment by Saam01** [comment]
@raphaelkross  Taxable ticket option is missing in kses-10 (Option is available in masters)
image

**Comment by Saam01** [comment]
I've finished testing this PR.

**Comment by hoseinrafiei** [comment]
> Taxable ticket option is missing in kses-10 (Option is available in masters)

@Saam01 

I can see the taxable option. Please check it again and make sure that you have some tax records in Pricing menu.
Screen Shot 2022-05-22 at 18 23 22

Whatever the results is, please reply here.
Also please make sure that you tag the correct person in your reply.

**Comment by Saam01** [comment]
@hoseinrafiei It seems to be working fine for me also after resetting the data.

**Pull request merged**